charts/zalenium/values.yaml

```diff
@@ -156,8 +156,8 @@ ingress:
   # If your ingress host name is shared with multiple
   # applications, Enter a path starting with / (eg. /zalenium)
   # If your ingress host name is only for Zalenium,
-  # set the path as / (root)
-  path: /
+  # no need to set path
+  # path: /
```

Comment:
This was a default value

Reply:
Some ingress load balancers are not forwarding all requests when the path is set to `/`, it is expecting `/*`.  If path itself is not there, all requests are forwarded.

If you feel strongly about this, I'll set path in values.yaml to `/` and change only in my deployment